integration-tests/smallrye-stork-consul-registration/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
I'm not sure why you have your own wrapper here? We should avoid that as it's more burden when we update.

If you have a good reason for that, I'm all ears :).